assets/src/dashboard/components/inlineInputForm/index.js

```diff
@@ -87,18 +87,17 @@ const InlineInputForm = ({
   );
   return (
     <div ref={inputContainerRef}>
-      <Label htmlFor={id}>{label}</Label>
-      <TextInput
-        type="text"
-        id={id}
-        data-testid={'inline-input-form'}
+      <Label htmlFor={`${id}`}>{label}</Label>
+      <StyledInput
+        id={`${id}`}
+        data-testid="inline-input-form"
         value={newValue}
         onKeyDown={handleKeyPress}
         onChange={handleChange}
         placeholder={placeholder}
-        error={error}
+        hasError={Boolean(error)}
+        hint={error}
       />
-      {error && <ErrorText>{error}</ErrorText>}
```

Comment:
The new input has its own hint/error text. however, the old text was smaller than the new one (By 2 px). Decided to use the default.

However, we don't use the `error` right now on this element throughout our codebase, so I figured it's okay to remove since it doesn't change anything that the user sees.

Can add this back in if we hate it.

Reply:
Awesome! yeah this was in there because it used to be shared at one point by settings I think.